foundations/foundation-common/src/main/java/io/servicecomb/foundation/common/utils/ReflectUtils.java

```diff
@@ -55,6 +57,10 @@ public static void setField(Class<?> cls, Object instance, String fieldName, Obj
   public static Method findMethod(Class<?> cls, String methodName) {
     for (Method method : cls.getMethods()) {
       if (method.getName().equals(methodName)) {
+        if (method.isAnnotationPresent(ApiOperation.class) &&
+            method.getAnnotation(ApiOperation.class).hidden()) {
```

Comment:
not  so good  that associate swagger with reflection
you  can do this  in swagger generator project